h/static/scripts/controllers.coffee

```diff
@@ -170,45 +198,43 @@ class ViewerController
     loaded = []
 
     loadAnnotations = ->
-      if annotator.tool is 'highlight'
+      if annotationUI.tool is 'highlight'
         return unless auth.user
         query = user: auth.user
 
-      for p in annotator.providers
+      for p in crossframe.providers
         for e in p.entities when e not in loaded
           loaded.push e
-          store.SearchResource.get angular.extend(uri: e, query), (results) ->
-            annotator.loadAnnotations(results.rows)
+          r = store.SearchResource.get angular.extend(uri: e, query), (results) ->
```

Comment:
Ditto vis-à-vis injecting both the `store` and the `annotationMapper`. It seems like this happens because the search resource is bolted onto annotation persistence. Perhaps introduce a distinct `searchMapper` class which wraps the `store` but only exposes a search API.